cmd/server.go

```diff
@@ -109,7 +112,56 @@ func produceServerCmdRunFunc(o serverOperator, action int) cmdRunFunc {
 				log.Fatalf("Removing Server failed: %s", err)
 			}
 		}
+	case serverCreateAction:
+		return func(cmd *cobra.Command, args []string) {
+			ctx := context.Background()
+			cServer, err := client.CreateServer(ctx, gsclient.ServerCreateRequest{
+				Name:   servernameFlag,
+				Cores:  cpuFlag,
+				Memory: memFlag,
+			})
+			if err != nil {
+				log.Fatal("Create server has failed with error", err)
+			}
+			log.WithFields(log.Fields{
+				"server_uuid": cServer.ObjectUUID,
+			}).Info("Server successfully created")
 
+			if templateFlag != "" {
+				template, _ := client.GetTemplateByName(ctx, templateFlag)
+				cStorage, err := client.CreateStorage(ctx, gsclient.StorageCreateRequest{
+					Name:        string(servernameFlag),
+					Capacity:    storageFlag,
+					StorageType: gsclient.DefaultStorageType,
+					Template: &gsclient.StorageTemplate{
+						TemplateUUID: template.Properties.ObjectUUID,
+						Password:     passwordFlag,
+						PasswordType: gsclient.PlainPasswordType,
+						Hostname:     hostnameFlag,
+					},
+					Labels: []string{"label"},
+				})
+
+				client.CreateServerStorage(
+					ctx,
+					cServer.ObjectUUID,
+					gsclient.ServerStorageRelationCreateRequest{
+						ObjectUUID: cStorage.ObjectUUID,
+						BootDevice: true,
+					})
+
+				if err != nil {
+					log.Error("Create storage has failed with error", err)
+					return
```

Comment:
Again, for error handling, make sure to give non-zero exit code when bailing out. Take a look at the other commands.

Reply:
Oh again? I'm guilty, will change it! 😄